tests/framework/installer/ceph_manifests_v1.1.go

```diff
@@ -1698,6 +1698,41 @@ spec:
                    path:  ` + removalDir
 }
 
+// GetCleanupAssertionPod asserts that the dataDirHostPath is empty
+func (m *CephManifestsV1_1) GetCleanupVerificationPod(node, hostPathDir string) string {
```

Comment:
You can just return the empty string for the 1.1 implementation since 1.1 didn't have a cleanup pod. The upgrade test shouldn't be calling this method anyway.

Reply:
GetCleanupPod should also return empty string, good catch.